jest.config.js

```diff
@@ -6,11 +6,10 @@ module.exports = {
   coverageDirectory: 'coverage',
   moduleNameMapper: {
     '\\.css$': 'identity-obj-proxy',
-    svg4everybody: 'identity-obj-proxy',
     'react-portal': 'identity-obj-proxy',
     shortid: 'identity-obj-proxy',
     '.*\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$':
-      'identity-obj-proxy',
+      '<rootDir>/jest/mocks/fileMock.js',
```

Comment:
lesson learned -- test errors that look something like
```
Error: Uncaught [TypeError: symbol is not a function]
```
as was this case, are usually due to files or functions that are loaded through webpack and thus `jest` can't natively handle. Will almost always need some extra mocking

Reply:
Is there a place to record error cases like these and various approaches to solve?

Reply:
good Q! do you think one of our existing FE wiki pages would make sense? if not could make a new one like "Jest debugging"